bgw-core/src/main/kotlin/tools/aqua/bgw/builder/UINodeBuilder.kt

```diff
@@ -112,6 +115,9 @@ internal class UINodeBuilder {
 
 			node.textProperty().bindTextProperty(textField)
 			node.promptText = textField.prompt
+			textField.onUpdatePrompt = {
```

Comment:
A ``guiListener`` should be set on an observable property instead of setting a function reference.